LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017-2018 Dash Core Group, Inc.
+Copyright (c) 2019 Dash Core Group, Inc.
```

Comment:
I believe this should be 2017-2019 as this work as started in 2017, continued thru 2018 and finally finished and released in 2019.